### PR TITLE
Swik 2058 bug fix

### PR DESCRIPTION
**Pull request by Sole-Valero**
Recent ImportModal needs an extra state to control it is showing an error to the user.
This fix adds it in order to prevent possible problems. Also, it changes the way we indicate which colors we want to use for buttons, in order to avoid some warnings.

### CODE REVIEW

File: components/Import/ImportModal.js [hunk 1]
```diff
@@ -107,24 +108,22 @@ class Import extends React.Component {
...
-
-        if(!this.state.cancelled && !this.state.accepted){ //user clicks outside to cancel without pressing upload button
-            this.setState({
-                cancelled:true
-            });
-
-            if(this.props.ImportStore.fileReadyForUpload){
-                this.context.executeAction(importCanceled, {});
+        if(!this.state.onShowError){
+            if(!this.state.cancelled && !this.state.accepted){ //user clicks outside to cancel without pressing upload button
```
Comment (pierorex) [inline components/Import/ImportModal.js]: For me, when clicking outside the import modal, it doesn't close it so I'm not able to check if this part works

**Comment by Sole-Valero** [comment]
But if you press esc, you will exit.

**Comment by abijames** [comment]
I have tested this and it is ok to merge @kadevgraaf

**Comment by kadevgraaf** [comment]
Thanks!

**Pull request merged**